frontend/src/ui/D3Visualizations/Treemap.tsx

```diff
@@ -54,43 +74,83 @@ export default function Treemap({ height, width, data }: TreemapProps) {
       .append('g')
       .attr('transform', d => `translate(${d.x0},${d.y0})`);
 
-    // lighten colors
-    const fader = (color: string): string =>
-      d3.interpolateRgb(color, white)(0.3);
-
     // select color within d3.schemeSet3 (contains array of 12 colors)
-    const colorScale = d3.scaleOrdinal(d3.schemeSet3.map(fader));
+    const colorScale = d3
+      .scaleQuantile()
```

Comment:
scaleQuantile is a generic function with a type parameter for the type of the range. It defaults to `number`, which is why you're getting the typescript error. I guess the compiler isn't smart enough to infer the value. Anyway, I am able to fix the error by replacing this line with

`.scaleQuantile<string>`